Bondora.lua

```diff
@@ -101,8 +102,10 @@ function RefreshAccount (account, since)
 end
 
 function EndSession ()
-  local url = baseUrl .. "/authorize/logout/"
+  local url = baseUrlLocale .. "/authorize/logout/"
   MM.printStatus("Logout: " .. url)
   connection:get(url)
   return nil
 end
+
+-- SIGNATURE: MCwCFQCTUbXmmBzwn+HcaBSuZ5cJgbiBuwITBD7d01qCIvvJxquVlzDa4JMDFA==
```

Comment:
@phaus Thanks for the PR. I'm not familiar with this signature concept. Is this the signature of the file's content? Can you link me to some documentation?

Reply:
Oh, sorry. Now I understand.

This signature is actually added by MoneyMoney themselves. The process is basically manually sending the extension file to MoneyMoney, they will sign the extension and host it on [their website](https://moneymoney-app.com/extensions/).

So you can remove the signature here.

Reply:
MoneyMoney has to signed the extension in order to have run in all Version (e.g. the AppStore Version). See https://twitter.com/moneymoneyapp/status/1244676350490918914
I assume, that the Signature changes, if the content of the script does change.